src/proxy/http/balance.rs

```diff
@@ -188,3 +650,164 @@ pub mod weight {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use proxy::{
+        http::router::rt,
+        pending,
+        resolve::{self, Resolution, Resolve},
+    };
+
+    fn assert_make<A, T: Clone>(_: A)
```

Comment:
Are these from the original bounds debugging? Looks like you can remove some of this if these tests still apply.

Reply:
yes, but i thought it was better to leave the tests to guard against problems in the future?

Reply:
Ah my fault the test is on the actual test signatures. I was seeing these as empty tests. All good -- definitely good to keep these.